pensieve/cli.py

```diff
@@ -96,22 +84,54 @@ def run(project_id, dataset_id, date, experiment_slug, dry_run, config_file):
         # run analysis for specific experiment
         active_experiments = active_experiments.with_slug(experiment_slug)
 
+    # get experiment-specific external configs
+    external_configs = ExternalConfigCollection.from_github_repo()
+
     # calculate metrics for experiments and write to BigQuery
     for experiment in active_experiments.experiments:
         spec = default_spec_for_experiment(experiment)
+
         if config_file:
             custom_spec = AnalysisSpec.from_dict(toml.load(config_file))
             spec.merge(custom_spec)
+
+        external_experiment_config = external_configs.spec_for_experiment(experiment.normandy_slug)
+
+        if external_experiment_config:
+            spec.merge(external_experiment_config)
+
         config = spec.resolve(experiment)
         Analysis(project_id, dataset_id, config).run(date, dry_run=dry_run)
 
+    if rerun_config_changed:
+        rerun_config_changed(project_id, dataset_id, dry_run)
 
-@cli.command()
-@experiment_slug_option
+
+@cli.command("run")
 @project_id_option
 @dataset_id_option
+@click.option(
+    "--date",
+    type=ClickDate(),
+    help="Date for which experiments should be analyzed",
+    metavar="YYYY-MM-DD",
+    required=True,
+)
+@experiment_slug_option
 @dry_run_option
 @secret_config_file_option
+@click.option(
+    "--rerun_config_changed",
+    "--rerun-config-changed",
+    help="Re-run experiments if their config changed.",
+)
```

Comment:
This seems like a separate action to me and I think we should ask users to run the command again if they want to do two things; is there a reason to roll them together?

Reply:
It'd be used by the Airflow task. If the config gets changed but for some reason the Ci doesn't automatically trigger a re-run then the Airflow task would re-run updated experiments if this flag is set.

Reply:
Another way to do this instead of adding a new flag would be to have a `jetstream ci` command that chains them.

Reply:
True, we could just call `jetstream run` followed by `jetstream rerun_config_changed` in `entrypoint`